PiWeb.Api.Training.OidcAuth/PiWeb.Api.Training.OidcAuth.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>net6.0-windows</TargetFramework>
+    <Nullable>enable</Nullable>
+    <UseWPF>true</UseWPF>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Remove="Resources\extract_cloud_login_inputs.js" />

```

Comment:
In der ItemGroup machst du ein Remove, nur um es darunter wieder hinzuzufügen. Brauch es das Remove vorher?

Reply:
Ich dachte, es wuerde gebraucht. Ich habe es nocheinmal ausprobiert und nein, es wird nicht gebraucht. -> werde ich entfernen.